zerver/openapi/curl_param_value_generators.py

```diff
@@ -127,7 +128,7 @@ def add_emoji_to_message() -> Dict[str, object]:
     user_profile = helpers.example_user("iago")
 
     # from OpenAPI format data in zulip.yaml
-    message_id = 43
+    message_id = get_last_message_id()
```

Comment:
That 43 seems to be hardcoded and intended to correspond to a hardcoded value in `zulip.yaml`. So this seems possibly incorrect?

Reply:
Yes, I was wrong about this. I addressed this in #20723, so should no longer be an issue for this PR. Thank you for pointing it out! :)